### DISCUSSION TITLE
Try-it Feature for APIs

### DISCUSSION THREAD

**Discussion by malathikrishnap** [rohit-gohri/redocusaurus#328] [Q&A]
Hi there, does Redocusaurus provide the "Try-it" feature for the APIs?
I've successfully integrated Redocusaurus 2.0 with my Docusaurus 3.1 project. However, I didn't get the "Try-it" option. 
How do I get that option?

**Comment by rohit-gohri** [comment]
Not in scope of this package. See https://github.com/rohit-gohri/redocusaurus/issues/24
